src/traces/box/attributes.js

```diff
@@ -312,6 +312,8 @@ module.exports = {
             {arrayOk: false, editType: 'plot'}),
         opacity: extendFlat({}, scatterMarkerAttrs.opacity,
             {arrayOk: false, dflt: 1, editType: 'style'}),
+        angle: extendFlat({}, scatterMarkerAttrs.angle,
+            {arrayOk: false, editType: 'calc'}), // TODO: arrayOk angle might be useful for certain box & violin graphs, No?
```

Comment:
I suppose one could imagine a use for `arrayOk` in any of these attributes... but `angle` is probably one of the last ones you'd want, and we don't allow it even for the more likely ones such as `color`. So I wouldn't bother leaving this as a TODO.